### ISSUE TITLE
Sound Desync

### ISSUE THREAD

**Issue by rdanbrook** [rdanbrook/nestopia#34]
I keep getting messages about this. A lot of people claim to experience sound desync issues from win32 version 1.37 onwards. I can't reproduce this, so it's impossible for me to fix or even look at it at this point.

If anyone else can reproduce this problem and give more details including hardware,  operating system, etc, it would help greatly in discovering the source of this problem. Also, feel free to point me towards any forum posts or mailing lists that may describe the problem. 

If you experience this problem on another platform, it would also be helpful to know. I have never experienced sound desync on any platform personally.

My current setup which does not exhibit the problem:
Windows 7 x64
Intel Core i7
Intel HD Audio

**Comment by SammyTheFox** [comment]
Did some testing with the Nestopia core in RetroArch and I've found that NTSC games do not have this issue (as far as I've tested), but PAL games seem to be affected by the sound desync bug.  I'm not sure if it's just a coincidence or what, but does RetroArch's Nestopia core handle the sound any differently than the stand-alone emulator?

**Comment by rdanbrook** [comment]
I basically never use PAL games, but when I was testing with them previously I did notice some pretty nasty issues. I can't remember if it was libao or SDL audio that messes up in the unix build, but I can double check later in the day. It may be a problem in RetroArch due to the way it syncs audio and video frames. I chose to sync to sound in my own build, but RetroArch dynamically resamples sound to sync to video I believe... could be wrong though. The core audio is exactly the same for every different build.
